sklearn/cluster/_spectral.py

```diff
@@ -301,15 +301,15 @@ class SpectralClustering(ClusterMixin, BaseEstimator):
 
     Parameters
     ----------
-    n_clusters : integer, optional
+    n_clusters : integer, default=8
         The dimension of the projection subspace.
 
     eigen_solver : {None, 'arpack', 'lobpcg', or 'amg'}
```

Comment:
If possible, change also this one by `{'arpack', 'lobpcg', 'amg'}, default=None` (since the default is `None`).

Thanks!

Reply:
ok